wg-ecosystem/meeting-notes/2020-10-29.md

```diff
@@ -0,0 +1,30 @@
+# Ecosystem WG
+
+Thursday October 29, 2020 (9:00 AM - 10:00 AM PT)
+
+## Attendees
+
+* @erickzhao
+* @malept
+* @toinane
+* @vhashimotoo
+
+## Agenda
+
+- @vhashimotoo stepping down from Electron governance.
+  - Vlad is leaving his role in the working group and resigning as chair. He won't be available on Slack anymore,
+  but will still be active on the Electron Discord server (and will still maintain his admin role there).
+  - Work to hand over:
+    - [electron-i18n#785](https://github.com/electron/i18n/pull/785): Support for the docs to handle multiple major versions of Electron.
+    This PR works, but build times increase massively (from ~5 minutes to ~40 minutes). His suggested alter
```

Comment:
This sentence trails off :smile: 

```suggestion
    This PR works, but build times increase massively (from ~5 minutes to ~40 minutes). His suggested alter
```

Reply:
Good point, I'll fix that. The alternative IIRC was to do the multi-version build on the website's side I believe.